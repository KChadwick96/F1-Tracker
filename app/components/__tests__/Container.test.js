import React from 'react';
import { Text } from 'react-native';
import { shallow } from 'enzyme';

import Container from '../Container';

it('renders its children', () => {
  const wrapper = shallow(
    <Container>
      <Text>Hello</Text>
      <Text>There!</Text>
    </Container>
  );
  expect(wrapper.find(Text).length).toEqual(2);
  expect(wrapper).toMatchSnapshot();
});
