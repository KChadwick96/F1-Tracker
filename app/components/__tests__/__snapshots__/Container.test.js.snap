// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders its children 1`] = `
<Component
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Hello
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    There!
  </Text>
</Component>
`;
